### PR TITLE
Mission loader

### PR DESCRIPTION
**Pull request by privat**
Load and display the Nit missions from https://github.com/nitlang/nitutorial

### CODE REVIEW

File: src/db_loader.nit [hunk 1]
```diff
@@ -69,7 +69,7 @@ for player in players do
 	# load some statuses
 	for mission in config.missions.find_all do
 		var status = new MissionStatus(mission, player, mission.track)
-		if mission.is_unlocked_for_player(config, player) or 100.rand > 25 then
+		if mission.is_unlocked_for_player(config, player) or 100.rand > 25 or mission.parents.is_empty then
```
Comment (Morriar) [inline src/db_loader.nit]: This check is already done in `is_unlocked_for_player`:

https://github.com/privat/Missions/blob/3daa2d1a0bcd987343c3cc22cc7f3ff9eb4c657b/src/model/status.nit#L66

File: src/db_loader.nit [hunk 2]
```diff
@@ -51,6 +53,10 @@ for i in [1..10] do
 	end
 end
 
+var t = new Track("Nit", "Learn Nit")
+config.tracks.save t
+t.load_missions(config, "tracks/nit")
```
Comment (Morriar) [inline src/db_loader.nit:58]: Can you change it as an option so my deploy script will not explode?

**Comment by Morriar** [comment]
I like this one, some changes are needed before a merge

File: src/db_loader.nit (hunk 2 above)
Comment (privat) [inline src/db_loader.nit:58]: what kind of option? What kind of explosion?

Comment (Morriar) [inline src/db_loader.nit:58]: I have some work to do on options,. this can wait

**Comment by privat** [comment]
reroll: removed useless `mission.parents.is_empty`.  but did nothing for the explosion/config

**Comment by privat** [comment]
test this please

**Pull request merged**